rmw_fastrtps_shared_cpp/src/rmw_take.cpp

```diff
@@ -101,6 +103,7 @@ _take(
       if (sinfo.valid_data) {
         if (message_info) {
           _assign_message_info(identifier, message_info, &sinfo);
+          fabric_functions::fabric_logger(message_info, subscription, "FastRTPS");
```

Comment:
This is inside `take` whereas with eCAL, it's in `take_with_info`. We should definitely have them at least in the same function.

Reply:
Unfortunately, we can't get the `message_info->source_timestamp` from `take_with_info` in FastRTPS.  
message_info is assigned by this line which is  
`_assign_message_info(identifier, message_info, &sinfo);`  in `_take`.
And `take_with_info` calls `_take`. Therefore, message_info is empty in `take_with_info`.

Reply:
Thanks for the info!